lisa/sut_orchestrator/baremetal/cluster/pxe.py

```diff
@@ -100,7 +106,56 @@ def _connect(self) -> None:
         process.wait_output("\n", timeout=1, interval=0.1)
 
         self._process = process
-        self._log.debug("connected to serial console: {serial_port}")
+        self._log.debug(f"connected to serial console: {serial_port}")
+
+
+class Ip9285StartStop(features.StartStop):
+    def __init__(
+        self, settings: FeatureSettings, node: Node, platform: Platform
+    ) -> None:
+        super().__init__(settings, node, platform)
+
+    def _initialize(self, *args: Any, **kwargs: Any) -> None:
+        super()._initialize(*args, **kwargs)
+
+        context = get_node_context(self._node)
+        pxe_cluster = cast(schema.PxeCluster, context.cluster)
+        assert pxe_cluster.start_stop, "start_stop is not defined"
+
+        ip_power_runbook = pxe_cluster.start_stop.get_extended_runbook(schema.Ip9285)
+
+        self._request_cmd = (
+            f"http://{ip_power_runbook.host}/set.cmd?"
+            f"user={ip_power_runbook.username}+pass="
+            f"{ip_power_runbook.password}+cmd="
+            f"setpower+P6{ip_power_runbook.ctrl_port}"
+        )
+
+    def _set_ip_power(self, power_cmd: str) -> None:
+        response = requests.get(power_cmd, timeout=REQUEST_TIMEOUT)
+        response.raise_for_status()
+        self._log.debug("Command {power_cmd} done in set_ip_power")
+
+    def _stop(
+        self,
+        wait: bool = True,
+        state: features.StopState = features.StopState.Shutdown,
+    ) -> None:
+        request_off = f"{self._request_cmd}=0"
+        self._set_ip_power(request_off)
+        # To make sure power-off is fully settled down, it is recommended
+        # to wait a short time before power-on is triggered. Local tests showed
+        # 1s is actually good enough to get a stable power-off status, here
```

Comment:
request timeout is not used here, so the comment shouldn't mention it to avoid confusing.